### PR TITLE
Setup project with webpack and linters

### PR DESCRIPTION
**Pull request by olawalecoder**
Hi there
This task is about creating a list structure for a to-do-list app and I have done the following::heavy_check_mark: :heavy_check_mark: :heavy_check_mark: 

1. Set up a new project with webpack. 

2. Create an index.html file and write HTML markup there. Create an empty To-Do List placeholder. The index.html file must be set as a template using the HTML Webpack Plugin.white_check_mark

3. Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
- description [string].white_check_mark
- completed [bool].white_check_mark
- index: [number].white_check_mark

4. Write a function to iterate over the tasks array and populate an HTML list item element for each task.white_check_mark.
And the other required tasks like:
- Removing all hardcoded items from the tasks array.
- Creating a new JavaScript file for new functionality.
- Implementing a function for adding a new task (add a new element to the array).
- Implementing a function for deleting a task (remove an element from the array).
- Implementing a function for editing task descriptions and so on.

### CODE REVIEW

File: src/Methods.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import storageManager from './Storage.js';
+import Todolist from './Todolist.js';
+import UserInterface from './UserInterface.js';
+
+export default class Methods {
```
Comment (Whoistolu) [inline src/Methods.js:5]: - [ ] [REQUIRED] One of the requirements states that a function for adding a new task (add a new element to the array) should be implemented.
- [ ] [REQUIRED] Kindly note that it is also a requirement to add a function for editing task descriptions is implemented.
- [ ] [REQUIREMENT]Kindly implement a function that deletes a task (remove an element from the array).

File: src/Storage.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export default class storageManager {
```
Comment (Whoistolu) [inline src/Storage.js:1]: Great job implementing this: Your new tasks have the property `completed` set to `false` and the property `index` set to the value of the new array length 👍🏽

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @olawalecoder 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
Hi @olawalecoder, 

Great work on making the changes requested by a previous reviewer 👏 👏🏼 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- [ ] Please make sure that all Linters result should be green ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nicupop729 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @olawalecoder, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

well-1

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**